code/particle/particle.cpp

```diff
@@ -146,6 +135,16 @@ namespace particle
 			return false;
 		}
 
+		// treat particles on lower detail levels as 'further away' for the purposes of culling
+		float adjusted_dist = vm_vec_dist(&Eye_position, &info->pos) * powf(2.5f, (NUM_DEFAULT_DETAIL_LEVELS - Detail.num_particles));
+		// treat bigger particles as 'closer'
+		adjusted_dist /= info->rad;
+		float cull_start_dist = 1000.f;
+		if (adjusted_dist > cull_start_dist) {
+			if (frand() > 1.0f / (log2(adjusted_dist) - (log2(cull_start_dist) - 1.0f)))
```

Comment:
Oh, wait.  If `cull_start_dist` is basically a constant, we should just have it be equal to log2(1000), with a comment and adjust the calculation below to have fewer steps (and make cull_start_dist a const or constexpr, too).  Compiler might already do it, but just in case.

Reply:
Ah, my bad, log2(x) - log2(y) does equal log(x/y) though! I misread!